pycontrails/models/cocipgrid/cocip_grid_params.py

```diff
@@ -136,3 +136,13 @@ class CocipGridParams(CocipParams):
     #: Attach contrail evolution data to :attr:`CocipGrid.contrail_list`. Requires
     #: substantial memory overhead.
     verbose_outputs_evolution: bool = False
+
+    #: Add additional metric of ATR20 and global yearly mean RF to model output.
+    #: These are not standard CoCiP outputs but based on the derivation used
+    #: in the first supplement to :cite:`yin_predicting_2023`. ATR20 is defined
+    #: as the average temperature response over a 20 year horizon.
+    compute_atr20: bool = False
+
+    #: Constant factor used to convert global- and year-mean RF, [:math:`W m^{-2}`],
+    #: to ATR20, [:math:`K`], given by :cite:`yin_predicting_2023`.
+    global_rf_to_atr20_factor: float = 0.0151
```

Comment:
Are these additions necessary? I think both parameters should be inherited from `CocipParams`.